README.md

```diff
@@ -21,6 +21,20 @@ Source for ORAS website and documentation
   </a>
 </div>
 
+# Development
+
+## Online one-click setup
+
+You can use Gitpod (a free, online, VS Code-like IDE) for contributing. With a single click, it will launch a workspace (for Docusaurus 2) and automatically:
+
+* clone the docusaurus repo.
```

Comment:
Are these steps needed? When I went through testing, it started the workspace, installed the dependencies and built and started the site